pkgs/by-name/op/opensplat/package.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  gcc12Stdenv,
+  stdenv,
+  cmake,
+  ninja,
+  fetchFromGitHub,
+  fetchpatch,
+  python3,
+  opencv,
+  nlohmann_json,
+  nanoflann,
+  glm,
+  cxxopts,
+  config,
+  # Upstream has rocm/hip support, too. anyone?
+  cudaSupport ? config.cudaSupport,
+  cudaPackages,
+  autoAddDriverRunpath,
+  cmakeFlags ? [ ], # There's some interesting flags like OPENSPLAT_BUILD_SIMPLE_TRAINER or OPENSPLAT_MAX_CUDA_COMPATIBILITY
```

Comment:
Why not to make such things more explicit instead?

If anything, we have  `overrideAttrs`.

Reply:
Ah, right. I only learned about `overrideAttrs` two days ago. ^^ Forgot to apply that knowledge here. Got rid of this line.
(It seems that `simple_trainer` is really an internal tool, and cuda compatibility seems to be handled by other options in nix.)